src/components/Quickstart/StickyHeaderObserver.tsx

```diff
@@ -95,7 +95,7 @@ const StickyHeaderObserver: React.FC = () => {
     }
   }, [$navBarInfo.height])
 
-  return <div ref={thisRef} className={styles.stickyBar} hidden={true} data-sticky />
+  return <div ref={thisRef} className={styles.stickyBar} hidden={true} data-sticky></div>
```

Comment:
Is this change necessary?

Reply:
Semantic change, just don't think divs should be self closing

Reply:
[P.S. this will be deleted in this PR anyways](https://github.com/smartcontractkit/documentation/pull/1584/files#diff-8e0d9c121aa3be61e192bb5dc7ff49965a968c54cb42463af7b6ef360a13344d)